hack/sync-forks.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#
+# This script synchronizes specified fork with the UPSTREAM commits.
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+tmpDir=$(mktemp -d)
```

Comment:
who deletes this?

Reply:
Should be using `$BASETMPDIR` or something under it -- `hack/lib/init.sh` cleans up after itself